sympy/assumptions/tests/test_query.py

```diff
@@ -595,13 +601,13 @@ def test_I():
     assert ask(Q.real(z)) is True
 
 
-
+@XFAIL  # definition for boundedness fixed! will be changed in next commit
 def test_bounded():
```

Comment:
Why is this XFAIL?

Reply:
It is because this test is based on incorrect design of Q.positive() and Q.negative(). I have fixed it, but it will make the diff quite large so I will make another PR for that.